test/errors.tests.js

```diff
@@ -28,11 +28,11 @@ describe('Errors', function() {
       const sanitizedError = errors.sanitizeErrorRequestData(error);
```

Comment:
do we want to update these variables as well?

Reply:
Updated.